src/widgets/forms/forms.component.tsx

```diff
@@ -68,6 +73,20 @@ const Forms: React.FC<any> = () => {
     setCompletedForms(completedForms);
   }, [allForms, encounters]);
 
+  React.useEffect(() => {
+    const filledForms = allForms.map(form => {
+      let checkedForm: Form;
+      completedForms.map(completeForm => {
+        if (completeForm.uuid === form.uuid) {
+          completeForm.checked = true;
+          checkedForm = completeForm;
+        }
+      });
+      return checkedForm ? checkedForm : form;
```

Comment:
Why not just

```suggestion
      completedForms.map(completeForm => {
        if (completeForm.uuid === form.uuid) {
          form.checked = true;
        }
      });
      return form;
```

Reply:
Oh, I see, the `form` and the `completedForm` objects may be different? It's a little confusing that we have multiple copies of forms floating around in this component. But don't worry about it, this is probably fine.